### ISSUE TITLE
[BUG] NumberFormatException: For input string: "0,00"

### ISSUE THREAD

**Issue by arphox** [ItsWagPvP/EconomyPlus#27]
**Describe the bug**
When creating a hologram to create a top list to show players' money top list with the command `/economyplus hologram`, an exception happens and the feature does not work.

Exception description:
```java
[20:17:17] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'economyplus' in plugin EconomyPlus v4.1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:172) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchCommand(CraftServer.java:906) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2389) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2200) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2181) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundChatPacket.a(ServerboundChatPacket.java:6) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1426) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:188) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1403) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1396) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1374) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1256) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:320) ~[purpur-1.18.2.jar:git-Purpur-1612]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NumberFormatException: For input string: "0,00"
	at jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054) ~[?:?]
	at jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110) ~[?:?]
	at java.lang.Double.parseDouble(Double.java:651) ~[?:?]
	at me.itswagpvp.economyplus.misc.Utils.format(Utils.java:109) ~[EconomyPlus.jar:?]
	at me.itswagpvp.economyplus.hooks.holograms.HolographicDisplays.createHologram(HolographicDisplays.java:59) ~[EconomyPlus.jar:?]
	at me.itswagpvp.economyplus.commands.Main.onCommand(Main.java:120) ~[EconomyPlus.jar:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	... 21 more
```

**Comment by ItsWagPvP** [comment]
looks like that you've used baltop pattern, replace the , with a .

**Comment by arphox** [comment]
> looks like that you've used baltop pattern, replace the , with a .

I don't really understand this as a friend of mine reported this issue.

The actual issue is that in `Utils.java:109`-110 you are doing double rounding wrong, doing it by string.format and double.parse instead of doing it with actual rounding (e.g. with the `DecimalFormat` class).

This has been proven by the fact that the bug persisted on our win 10 machine with Hungarian locale (where we use comma as a decimal separator) and it wasn't buggy on our linux server with some other English locale.

The problem in code stems from a wrong implementation of a wrong solution of rounding numbers.

**Comment by ItsWagPvP** [comment]
i will fix it in the next update

**Comment by ItsWagPvP** [comment]
Fixed in 4.3
